src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationNameCompletionProvider.DeclarationInfo.cs

```diff
@@ -251,13 +252,31 @@ internal struct NameDeclarationInfo
                 int position, CancellationToken cancellationToken, out NameDeclarationInfo result)
             {
                 result = IsFollowingTypeOrComma<VariableDeclarationSyntax>(token, semanticModel,
-                     v => v.Type,
-                     v => v.Parent is LocalDeclarationStatementSyntax l ? l.Modifiers : default(SyntaxTokenList?),
+                     typeSyntaxGetter: v => v.Type,
+                     modifierGetter: v =>
+                        v.Parent is LocalDeclarationStatementSyntax localDeclaration ? localDeclaration.Modifiers :
+                        v.Parent is UsingStatementSyntax ? default(SyntaxTokenList) :
+                        v.Parent is ForStatementSyntax ? default(SyntaxTokenList) : null as SyntaxTokenList?,
```

Comment:
why not ```default(SyntaxTokenList?)```

Reply:
I already described above: https://github.com/dotnet/roslyn/pull/26026#discussion_r180060423

Reply:
fixed in ad770dce343a57fa62c75368be125791cdc21ffb